### ISSUE TITLE
Mmusculus.v79 UCSC/NCBI chr name name mismatch.

### ISSUE THREAD

**Issue by mzytnicki** [Bioconductor/GenomeInfoDb#28]
It is probably a consequence of the patch of Issue #27 

    tmp <- genes(EnsDb.Mmusculus.v79)
    seqlevelsStyle(tmp) <- "UCSC"

This triggers an error (poorly translated from French):

    Error in (function (UCSC_chrom_info, assembly_accession, AssemblyUnits = NULL, identical(UCSC_chrom_info[compare_idx, chrom"], NCBI_chrom_info[compare_idx,  .... is not TRUE

I cannot pinpoint which chromosome name is wrong (sorry, I am still a novice), but the function should be: `.add_NCBI_cols_to_UCSC_chrom_info`

Any suggestion about where I should look for to find the culprit?

**Comment by maximilianh** [comment]
This could be related to the other ticket #27 about NCBI/UCSC chrom names where @hpages noticed something related. I reached out to NCBI and they told me where they got the "UCSC-style-ids" from. It was from patch4 so this file http://hgdownload.soe.ucsc.edu/gbdb/mm10/mm10Patch4/ which we only added as an annotation track, not as genome sequences and this file that was not intended to be used as a reference. We pointed NCBI now to the p6 file with the real sequence names under the main download directory https://hgdownload.cse.ucsc.edu/goldenPath/mm10/bigZip, so for p6 this is https://hgdownload.cse.ucsc.edu/goldenPath/mm10/bigZips/p6/mm10.p6.chrom.sizes and they say that they will update their UCSC-style-ids table ("assembly report") very soon. Sorry for the confusion, UCSC and NCBI should really communicate more about the sequence names, we'll do this in the future. We need to reach out to NCBI and Bioconductor when we update old assemblies and they will tell us which files they use from our site. I have more and more the impression that patches are not worth the effort for most analyses, updates to reference genomes cause a lot of trouble, but of course I do not know the full spectrum of genomics.
